quickstarts/microsoft.machinelearningservices/machine-learning-end-to-end-secure/modules/machinelearningcompute.bicep

```diff
@@ -0,0 +1,133 @@
+// Creates compute resources in the specified machine learning workspace
+// Includes Compute Instance, Compute Cluster and attached Azure Kubernetes Service compute types
+@description('Prefix for resource names')
+param prefix string
+
+@description('Azure Machine Learning workspace to create the compute resources in')
+param machineLearning string
+
+@description('Azure region of the deployment')
+param location string
+
+@description('Tags to add to the resources')
+param tags object
+
+@description('Resource ID of the compute subnet')
+param computeSubnetId string
+
+@description('Name of the Azure Kubernetes services resource')
+param aksName string
+
+@description('Resource ID of the Azure Kubernetes services resource')
+param aksSubnetId string
+
+@description('Resource ID of the Azure Kubernetes services resource')
+param amlComputePublicIp bool
+
+@description('VM size for the default CPU compute cluster')
+param cpuDefaultVmSize string = 'Standard_Ds3_v2'
+
+@description('VM size for the default GPU compute cluster')
+param gpuDefaultVmSize string = 'Standard_NC6'
```

Comment:
This size is throwing an error in the region - we need to either restrict the sizes, region or allow for setting the size

Reply:
It's due to missing region quota. I'll remove the gpu cluster configuraiton.

Reply:
Azure ML compute is hosted-on-behalf-of, and users require Azure ML compute specific quota. We do have defaults, but for specialized compute including NC6 these are zero by default. I've removed the GPU compute configuration completely with latest update.